packages/grub/latest-srpm-url.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
-docker run --rm amazonlinux:2 sh -c 'yum install -q -y yum-utils && yumdownloader -q --source --urls grub2 | grep ^http'
+cmd='dnf install -q -y --releasever=latest yum-utils && yumdownloader -q --releasever=latest --source --urls grub2'
```

Comment:
Will this make sure that the latest RPM repo will be used?

Reply:
I assume you're referring to Amazon Linux 2022's version locking? If so, yes, this will always pull the latest RPM due to `--releasever=latest`. I happened to still have an old container image around, so can provide some evidence as well. :-)

```
bash-5.1# yumdownloader -q --source --urls grub2
https://al2022-repos-us-west-2-9761ab97.s3.dualstack.us-west-2.amazonaws.com/core/guids/7121068131b337400594464d34e729a039727242067ddefbd6a8545886479c22/SRPMS/../../../../blobstore/aa41fdf9982b65a4c4dad5df5b49ba143b1710d60f82688221966f3c790c6c63/grub2-2.06-42.amzn2022.0.1.src.rpm
bash-5.1# yumdownloader -q --releasever=latest --source --urls grub2
https://al2022-repos-us-west-2-9761ab97.s3.dualstack.us-west-2.amazonaws.com/core/guids/3639f46a50c6c4978d36ecda0895b164e42fa86ff7fd8dc984898128901b9962/SRPMS/../../../../blobstore/aa41fdf9982b65a4c4dad5df5b49ba143b1710d60f82688221966f3c790c6c63/grub2-2.06-42.amzn2022.0.1.src.rpm
```

Note that here only the repo GUIDs differ.